### PR TITLE
MDEXP-311 Fix wrong job status when all UUIDs in the last batch not present in storages

### PR DESCRIPTION
**Pull request by andrii-novytskyi**
## Purpose
https://issues.folio.org/browse/MDEXP-311

## Approach
If the batch is last and no records are found in inventory or srs, add check if mrc file exist in file system, if exist - status CompletedWithErrors, if not - Failed.
Added unit test

### CODE REVIEW

**Review by Igor-Gorchakov — APPROVED** [review]

File: src/main/java/org/folio/service/manager/export/ExportManagerImpl.java [hunk 1]
```diff
@@ -203,19 +203,26 @@ private ExportResult getExportResult(AsyncResult<Object> asyncResult, ExportPayl
...
-        } else {
-          return ExportResult.completed();
-        }
+        return getProperExportResultIfBatchIsLast(exportPayload);
       }
       return ExportResult.inProgress();
     }
   }
 
+  private ExportResult getProperExportResultIfBatchIsLast(ExportPayload exportPayload) {
```
Comment (Igor-Gorchakov) [inline src/main/java/org/folio/service/manager/export/ExportManagerImpl.java:212]: Rename to getExportResultForLastBatch

File: src/main/java/org/folio/service/manager/export/ExportManagerImpl.java [hunk 2]
```diff
@@ -203,19 +203,26 @@ private ExportResult getExportResult(AsyncResult<Object> asyncResult, ExportPayl
...
+    if (exportPayload.getExportedRecordsNumber() == 0) {
+      if (vertx.fileSystem().existsBlocking(exportPayload.getFileExportDefinition().getSourcePath())) {
```
Comment (Igor-Gorchakov) [inline src/main/java/org/folio/service/manager/export/ExportManagerImpl.java:214]: Use FileStorage service to check the existence of file

File: src/main/java/org/folio/service/manager/export/ExportManagerImpl.java (hunk 1 above)
Comment (andrii-novytskyi) [reply src/main/java/org/folio/service/manager/export/ExportManagerImpl.java]: done

File: src/main/java/org/folio/service/manager/export/ExportManagerImpl.java (hunk 2 above)
Comment (andrii-novytskyi) [reply src/main/java/org/folio/service/manager/export/ExportManagerImpl.java]: done

**Review by IlliaDaliek — APPROVED** [review]

**Pull request merged**